docsource/buildinitclient.rst

```diff
@@ -74,26 +77,23 @@ This simple program initializes and calls a method on the WSS client::
 
 .. code-block:: java
 
+    import java.net.URI;
     import java.net.URL;
-    import us.kbase.auth.AuthConfig;
-    import us.kbase.workspace.WorkspaceClient;
-    import us.kbase.auth.ConfigurableAuthService;
     import us.kbase.auth.AuthToken;
+    import us.kbase.auth.client.AuthClient;
+    import us.kbase.workspace.WorkspaceClient;
 
     public class TryWorkspaceClient {
 
         public static void main(String[] args) throws Exception {
-            String authUrl =
-                "https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login/";
-
-            ConfigurableAuthService authService = new ConfigurableAuthService(
-                    new AuthConfig().withKBaseAuthServerURL(new URL(authUrl));
+            final String authUrl = "https://appdev.kbase.us/services/auth/";
+            final AuthClient authcli = AuthClient.from(new URI(authUrl));
```

Comment:
why switch from `CI` to `appdev`?

Reply:
Because this documentation is targeted at 3rd app developers as well as kbase developers and they're much more likely to have an appdev/prod account than a ci account